lib/index.js

```diff
@@ -22,11 +22,15 @@ webFrame.setZoomLevelLimits(1, 1);
 
 const store_ = createStore(
   rootReducer,
-  applyMiddleware(
-    thunk,
-    plugins.middleware,
-    thunk,
-    effects
+  compose(
+    applyMiddleware(
+      thunk,
+      plugins.middleware,
+      thunk,
+      effects
+    ),
+    // Redux DevTools extension integration
+    window.devToolsExtension ? window.devToolsExtension() : f => f
```

Comment:
Not sure, I just used the recommended setup: https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup

I'm guessing `compose()` expects a function, so a noop function that just returns what it's given (`f => f`) is used when the Redux Devtools extension hasn't been installed yet.